src/Provider/Provider.Web/Startup.ConfigureServices.cs

```diff
@@ -28,9 +30,13 @@ public void ConfigureServices(IServiceCollection services)
         {
             services.AddIoC(_configuration);
 
+            var redis = ConnectionMultiplexer.Connect(_configuration.GetConnectionString("Redis"));
+            services.AddDataProtection().PersistKeysToStackExchangeRedis(redis, "DataProtection-Keys");
+
             // Routing has to come before adding Mvc
             services.AddRouting(opt =>
             {
+                opt.LowercaseUrls = true;
```

Comment:
Perhaps make this consistent across other websites

Reply:
don't want employer account codes to be lowercased on employer website.